services/metadactyl-clj/src/metadactyl/validation.clj

```diff
@@ -236,27 +235,27 @@
   (when (and (contains? persistence/param-output-types param-type)
              (blank? default-value)
              (or (not visible) implicit))
-    (throw+ {:error_code cc-errs/ERR_BAD_OR_MISSING_FIELD
-             :message    "Hidden output parameters must define a default value."
-             :parameter  parameter})))
+    (throw+ {:type   :clojure-commons.exception/missing-request-field
+             :error  "Hidden output parameters must define a default value."
+             :parameter parameter})))
```

Comment:
This exception type (and probably the one above and the one below) should probably be a `bad-request-field`.